ql/indexes/ibor/tona.hpp

```diff
@@ -0,0 +1,50 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
+Copyright (C) 2016 Fabrice Lecuyer
```

Comment:
This should read 2022 and attribute copyright either to you or your company (depending on how you're contributing)

Reply:
I did.